docs/specs.md

```diff
@@ -164,8 +164,9 @@ and outputs. No html, markdown, or ansi will be rendered.
 * `"simple_format"` (default for top level output) - This is very
```

Comment:
Can we put the simple_format bullet point below the HTML one?

Reply:
Done